system-addon/lib/TopStoriesFeed.jsm

```diff
@@ -7,13 +7,15 @@ const {classes: Cc, interfaces: Ci, utils: Cu} = Components;
 Cu.import("resource://gre/modules/XPCOMUtils.jsm");
 Cu.import("resource://gre/modules/Services.jsm");
 Cu.import("resource://gre/modules/NewTabUtils.jsm");
+Cu.import("resource://gre/modules/Console.jsm");
```

Comment:
Probably a left-over and can be removed?

Reply:
oopsies!